libs/ledger/tests/chain/main_chain_service_tests.cpp

```diff
@@ -329,16 +462,15 @@ TEST_F(MainChainServiceTests, CheckWaitingToFullfilResponse)
   EXPECT_CALL(rpc_client_, TimeTravel(other1_, GetGenesisDigest()))
       .WillOnce(Return(TraveloguePromise{promise}));
 
-  Tick(State::SYNCHRONISING, State::START_SYNC_WITH_PEER);
```

Comment:
While I know you abhor duplication, being able to quickly comment stages in the pipeline is very easy with the old structure. Let’s discuss over tea in the office